click/base.js

```diff
@@ -3,3 +3,24 @@ console.log("Sanity Check: JS is working!");
 $(document).ready(function(){
 
 })
+
+//
+// 4. click
+//
+// Challenge: Can you create a list of all the phrases that you click?
+// Stretch: In addition to listing the phrase I clicked, can you include a timestamp?
+
+$("span").on("click", function handleClick(event) {
+  $("ul").append("<li>" + $(this).text() + "</li>");
+});
+
+
+$("span").on("click", function handleClick(event) {
+  $("ul").append("<li>" + $(this).text() + " at " + event.timeStamp + "</li>");
+});
+
```

Comment:
You need to move the $('span') logic inside the document ready function, otherwise the spans in question may not be loaded on the page when your script tries to attach the event listeners